app-frontend/src/app/pages/library/scenes/detail/detail.html

```diff
@@ -10,7 +10,9 @@ <h1 class="h3 page-title">
     <rf-map-container class="map-preview-container"
                       map-id="scene"
                       options="$ctrl.mapOptions"></rf-map-container>
-    <rf-scene-detail scene="$ctrl.scene"></rf-scene-detail>
+    <rf-scene-detail scene="$ctrl.scene"
```

Comment:
I think this needs some buffering in the detail view, otherwise it can get kind of cramped

![detail-view-needs-buffering](https://cloud.githubusercontent.com/assets/898060/21510531/75d009d0-cc62-11e6-8696-2c5e0bc7e717.png)